### PR TITLE
feat: Add curl executor.

### PR DESCRIPTION
**Pull request by nicufk**
This PR is related to [issue](https://github.com/kubeshop/kubtest/issues/52) 

## Fixes

-

## Checklist

- [ ] tested locally
- [ ] added new dependencies
- [ ] updated the docs
- [ ] added a test

### CODE REVIEW

File: kubtest/charts/kubtest/values.yaml [hunk 1]
```diff
@@ -143,6 +143,22 @@ cypress-executor:
...
+  image:
+    repository: kubeshop/kubtest-curl-executor
+    pullPolicy: Always
+    # Overrides the image tag whose default is the chart appVersion.
+    # TODO we should stick to static version
+    tag: "latest"
+  service:
+    type: NodePort # for working with real k8s we should use "ClusterIP" type.
+    port: 8086
+  mongoDSN: "mongodb://kubtest-mongodb:27017"
```
Comment (exu) [inline kubtest/charts/kubtest/values.yaml:159]: is it using mongo ?

**Review by exu — APPROVED** [review]

File: kubtest/charts/kubtest/values.yaml (hunk 1 above)
Comment (nicufk) [reply kubtest/charts/kubtest/values.yaml:159]: Right now it is with the in memory storage, but I want to make it to use mongo db if we are going to make it public.

Comment (olensmar) [reply kubtest/charts/kubtest/values.yaml:159]: doesn't the api-server write the test results returned by executors to mongo? does the executor need to communicate directly with mongo also?

Comment (nicufk) [reply kubtest/charts/kubtest/values.yaml:159]: Executors use also storage(mongo) when it receives an execution request, executor saves the execution to the storage with status "queued", then the worker is processing it, it changes status to "pending" and after the test is done it updates the status.

Comment (olensmar) [reply kubtest/charts/kubtest/values.yaml:159]: thanks @nicufk - perhaps this is something we should provide an abstraction for at some point - so we don't depend on mongo for this in each executor...

**Review by olensmar — APPROVED** [review]

**Pull request merged**